### PR TITLE
Add stack integration test for Node 12 templates

### PR DESCRIPTION
**Pull request by aahung**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aahung** [comment]
SQS template cannot be deployed due to some error. To be investigated. So SQS template does not have any integration tests for now.

File: nodejs12.x/cookiecutter-aws-sam-hello-nodejs/{{cookiecutter.project_name}}/hello-world/tests/integration/test-api-gateway.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+"use strict";
+
+const chai = require("chai");
+const AWS = require("aws-sdk");
+const https = require("https");
+const expect = chai.expect;
+
+const getStackName = () => {
```
Comment (c2tarun) [inline nodejs12.x/cookiecutter-aws-sam-hello-nodejs/{{cookiecutter.project_name}}/hello-world/tests/integration/test-api-gateway.js:8]: documentation.

File: nodejs12.x/cookiecutter-aws-sam-step-functions-sample-app/{{cookiecutter.project_name}}/integration-tests/test-state-machine.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+const chai = require("chai");
+const AWS = require("aws-sdk");
+const uuid = require("uuid");
+const expect = chai.expect;
+
+const sleep = (secs) =>
+  new Promise((resolve) => setTimeout(resolve, 1000 * secs));
+
+const getAndVerifyStackName = async () => {
```
Comment (c2tarun) [inline nodejs12.x/cookiecutter-aws-sam-step-functions-sample-app/{{cookiecutter.project_name}}/integration-tests/test-state-machine.js:11]: documentation

**Review by c2tarun — CHANGES_REQUESTED** [review]
Add documentation to the methods missing documentation.

**Comment by aahung** [comment]
> Add documentation to the methods missing documentation.

All resolved.

**Review by hoffa — APPROVED** [review]

**Review by c2tarun — APPROVED** [review]

**Pull request merged**